theforeman.org/yaml/jobs/release/pulpcore-packaging-rpm-release.yaml

```diff
@@ -26,3 +26,31 @@
     empty: ''
     version:
       !include: ../../includes/pulpcore_versions.yaml.inc
+
+- job-template:
+    name: 'pulpcore-packaging-rpm-copr-{version}-release'
+    project-type: pipeline
+    sandbox: true
+    concurrent: false
+    properties:
+      - github:
+          url: https://github.com/thepulpcore/pulpcore-packaging
+    triggers:
+        - github
+    dsl:
+      !include-raw:
+        - 'pipelines/vars/pulpcore/{version}.groovy'
+        - 'pipelines/release/pulpcore_packaging_rpm_copr_release.groovy{empty}'
+        - 'pipelines/lib/git.groovy{empty}'
+        - 'pipelines/lib/ansible.groovy{empty}'
+        - 'pipelines/lib/obal.groovy{empty}'
+        - 'pipelines/lib/packaging.groovy{empty}'
+        - 'pipelines/lib/foreman_infra.groovy{empty}'
+
+- project:
+    name: pulpcore-packaging-rpm-copr
+    jobs:
+      - 'pulpcore-packaging-rpm-copr-{version}-release'
+    empty: ''
+    version:
+      - '3.28'
```

Comment:
@Odilhao Do you want this to be nightly then as well?

Reply:
Yes pls, we can later add 3.28